web/src/components/onboarding-modal/onboarding-modal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { Button } from '../ui/ui';
+import { PlayOutlineGreenIcon, MicIcon, SkipIcon } from '../ui/icons';
+import Modal from '../modal/modal';
+import URLS from '../../urls';
+import { LocaleLink } from '../locale-helpers';
+import './onboarding-modal.css';
+
+const STEPS: Array<any> = [
+  {
+    title: 'Track your team and indvidual progress on the challenge dashboard',
+    img: './images/dashboard.png',
```

Comment:
These `img` values need to be wrapped in a `require` call to resolve to the correct path.

```
// ...
img: require('./images/dashboard.png'),
// ...
```